### PR TITLE
Deprecates zksync Goerli and adds new zkSync Sepolia testnet

### PR DESCRIPTION
**Pull request by githubdoramon**
Also updates some minor information on all zkSync networks

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
If you deprecate a chain -> you MUST add the status deprecated field
If you really want to reactivate a deprecated chain -> you MUST add the redFlags reusedChainId info

**Comment by githubdoramon** [comment]
> If you deprecate a chain -> you MUST add the status deprecated field If you really want to reactivate a deprecated chain -> you MUST add the redFlags reusedChainId info

Thanks @ligi . Addressed the comments

File: _data/chains/eip155-300.json [hunk 1]
```diff
@@ -1,17 +1,30 @@
 {
-  "name": "(deprecated) Optimism on GC",
-  "chain": "OGC",
-  "rpc": [],
+  "name": "zkSync Sepolia Mainnet",
```
Comment (Romsters) [inline _data/chains/eip155-300.json:2]:
```suggestion
  "name": "zkSync Sepolia Testnet",
```
?

Comment (githubdoramon) [reply _data/chains/eip155-300.json:2]: oh god. Thanks

**Comment by tilemanrenovations** [comment]
Hows it going for you Mr .  Digimon?

**Comment by 9844272456** [comment]
ในวันที่ พฤ. 7 ธ.ค. 2023 06:54 น. Travis C. Nelson Jr. <
***@***.***> เขียนว่า:

**Review by 9844272456 — APPROVED** [review]

**Comment by githubdoramon** [comment]
@ligi friendly nudge :) - Can you please review again based on the changes?

**Pull request merged**